cbindgen.toml

```diff
@@ -6,3 +6,8 @@
 language = "C"
 sys_includes = ["CoreFoundation/CoreFoundation.h"]
 no_includes = true
+
+[defines]
+"target_os=android" = "TARGET_OS_ANDROID"
```

Comment:
@i-schuetz Added this stuff after merge with master (cbindgen was complaining)

Reply:
This will need a fix. There are no Android headers / they shouldn't be in coepicore.h

Reply:
(I'll do it)

Reply:
cbindgen requires this entry because target_os config was used in the project
```
WARN: Missing `[defines]` entry for `target_os = "android"` in cbindgen config.

Reply:
Also, there is this:
```
ERROR: Cannot use fn coepi_core::Java_org_coepi_android_api_NativeApi_postReport (Parameter has an unsupported type.).
ERROR: Cannot use fn coepi_core::Java_org_coepi_android_api_NativeApi_testSendReceiveString (Parameter has an unsupported type.).
ERROR: Cannot use fn coepi_core::Java_org_coepi_android_api_NativeApi_bootstrapCore (Parameter has an unsupported type.).